pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/ModuleListingServlet.java

```diff
@@ -99,9 +100,15 @@ protected String getQuery(SlingHttpServletRequest request) {
 
         StringBuilder queryBuilder = null;
         if (repoParam != null) {
-            log.info("[" + ModuleListingServlet.class.getSimpleName() + "] contentTypeParam: " + contentTypeParam);
-            String repos = String.join(" | ", repoParam);
-            log.info("[" + ModuleListingServlet.class.getSimpleName() + "] repos: " + repos);
+            // encode if repo name contains space
+            String repos = Arrays.stream(repoParam)
+                    .map( repo -> {
+                        if (repo.contains(" ")) {
+                            return ISO9075.encode(repo);
```

Comment:
Just one query - what is the intent behind using ISO9875.encode rather than UTF-8?

Reply:
jackrabbit uses ISO9075.encode to handle the spaces.
https://www.mail-archive.com/users@jackrabbit.apache.org/msg05782.html

Reply:
That is an excellent find, bravo for being able to dig up this relatively obscure internet artifact.

Would you mind adding this link as a comment in the source code?

Also, do we need the `if (repo.contains(" "))` statement? Can we just encode the string unconditionally?